### ISSUE TITLE
[python-ray] fails to build 1.2.0 version

### ISSUE THREAD

**Issue by hubutui** [acxz/pkgbuilds#114]
1.2.0 is released, see also https://github.com/ray-project/ray/releases/tag/ray-1.2.0

https://github.com/acxz/pkgbuilds/blob/7eb795fd3c201b50b0e769facb393b0f07b36b5d/python-ray/PKGBUILD#L10

we could write simply `Apache` here, as this license is include in pkg `licenses`.

I try to build 1.1.0, but got errors:
```text
Extracting Bazel installation...
Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
DEBUG: /build/python-ray/src/ray-ray-1.1.0/bazel/ray_deps_setup.bzl:63:14: No implicit mirrors used because urls were explicitly provided
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
    currently loading: 
Analyzing: target //:ray_pkg (1 packages loaded, 0 targets configured)
Analyzing: target //:ray_pkg (19 packages loaded, 962 targets configured)
Analyzing: target //:ray_pkg (19 packages loaded, 962 targets configured)
Analyzing: target //:ray_pkg (19 packages loaded, 962 targets configured)
Analyzing: target //:ray_pkg (28 packages loaded, 1304 targets configured)
ERROR: /build/.cache/bazel/_bazel_builduser/6fe3cae9fa5ff97d4818b5c975921277/external/com_google_protobuf/BUILD:873:21: in proto_lang_toolchain rule @com_google_protobuf//:cc_toolchain: '@com_google_protobuf//:cc_toolchain' does not have mandatory provider 'ProtoInfo'.
INFO: Repository boost instantiated at:
  /build/python-ray/src/ray-ray-1.1.0/WORKSPACE:5:15: in <toplevel>
  /build/python-ray/src/ray-ray-1.1.0/bazel/ray_deps_setup.bzl:140:22: in ray_deps_setup
  /build/python-ray/src/ray-ray-1.1.0/bazel/ray_deps_setup.bzl:74:24: in auto_http_archive
Repository rule http_archive defined at:
  /build/.cache/bazel/_bazel_builduser/6fe3cae9fa5ff97d4818b5c975921277/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
INFO: Repository com_google_googletest instantiated at:
  /build/python-ray/src/ray-ray-1.1.0/WORKSPACE:5:15: in <toplevel>
  /build/python-ray/src/ray-ray-1.1.0/bazel/ray_deps_setup.bzl:169:22: in ray_deps_setup
  /build/python-ray/src/ray-ray-1.1.0/bazel/ray_deps_setup.bzl:74:24: in auto_http_archive
Repository rule http_archive defined at:
  /build/.cache/bazel/_bazel_builduser/6fe3cae9fa5ff97d4818b5c975921277/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
INFO: Repository arrow instantiated at:
  /build/python-ray/src/ray-ray-1.1.0/WORKSPACE:5:15: in <toplevel>
  /build/python-ray/src/ray-ray-1.1.0/bazel/ray_deps_setup.bzl:193:22: in ray_deps_setup
  /build/python-ray/src/ray-ray-1.1.0/bazel/ray_deps_setup.bzl:74:24: in auto_http_archive
Repository rule http_archive defined at:
  /build/.cache/bazel/_bazel_builduser/6fe3cae9fa5ff97d4818b5c975921277/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
INFO: Repository com_google_absl instantiated at:
  /build/python-ray/src/ray-ray-1.1.0/WORKSPACE:5:15: in <toplevel>
  /build/python-ray/src/ray-ray-1.1.0/bazel/ray_deps_setup.bzl:222:22: in ray_deps_setup
  /build/python-ray/src/ray-ray-1.1.0/bazel/ray_deps_setup.bzl:74:24: in auto_http_archive
Repository rule http_archive defined at:
  /build/.cache/bazel/_bazel_builduser/6fe3cae9fa5ff97d4818b5c975921277/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
INFO: Repository io_opencensus_cpp instantiated at:
  /build/python-ray/src/ray-ray-1.1.0/WORKSPACE:5:15: in <toplevel>
  /build/python-ray/src/ray-ray-1.1.0/bazel/ray_deps_setup.bzl:210:22: in ray_deps_setup
  /build/python-ray/src/ray-ray-1.1.0/bazel/ray_deps_setup.bzl:74:24: in auto_http_archive
Repository rule http_archive defined at:
  /build/.cache/bazel/_bazel_builduser/6fe3cae9fa5ff97d4818b5c975921277/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
INFO: Repository cython instantiated at:
  /build/python-ray/src/ray-ray-1.1.0/WORKSPACE:5:15: in <toplevel>
  /build/python-ray/src/ray-ray-1.1.0/bazel/ray_deps_setup.bzl:203:22: in ray_deps_setup
  /build/python-ray/src/ray-ray-1.1.0/bazel/ray_deps_setup.bzl:74:24: in auto_http_archive
Repository rule http_archive defined at:
  /build/.cache/bazel/_bazel_builduser/6fe3cae9fa5ff97d4818b5c975921277/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
ERROR: Analysis of target '//:ray_pkg' failed; build aborted: Analysis of target '@com_google_protobuf//:cc_toolchain' failed
INFO: Elapsed time: 16.885s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (40 packages loaded, 1990 targets configured)
FAILED: Build did NOT complete successfully (40 packages loaded, 1990 targets configured)
Traceback (most recent call last):
  File "/build/python-ray/src/ray-ray-1.1.0/python/setup.py", line 428, in <module>
    setuptools.setup(
  File "/usr/lib/python3.9/site-packages/setuptools/__init__.py", line 153, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python3.9/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/lib/python3.9/distutils/dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python3.9/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/usr/lib/python3.9/distutils/command/build.py", line 135, in run
    self.run_command(cmd_name)
  File "/usr/lib/python3.9/distutils/cmd.py", line 313, in run_command
    self.distribution.run_command(command)
  File "/usr/lib/python3.9/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "/build/python-ray/src/ray-ray-1.1.0/python/setup.py", line 421, in run
    return pip_run(self)
  File "/build/python-ray/src/ray-ray-1.1.0/python/setup.py", line 338, in pip_run
    build(True, BUILD_JAVA)
  File "/build/python-ray/src/ray-ray-1.1.0/python/setup.py", line 295, in build
    return bazel_invoke(
  File "/build/python-ray/src/ray-ray-1.1.0/python/setup.py", line 181, in bazel_invoke
    result = invoker([cmd] + cmdline, *args, **kwargs)
  File "/usr/lib/python3.9/subprocess.py", line 373, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['bazel', 'build', '--verbose_failures', '--', '//:ray_pkg']' returned non-zero exit status 1.
```
any idea to fix it?

**Comment by acxz** [comment]
@hubutui just updated it, you'll probably still encounter the same error tho. See upstream issue: https://github.com/ray-project/ray/issues/14117#issuecomment-780106540

**Comment by acxz** [comment]
Closed with the upgrade to 1.3.0 and the patch: https://github.com/ray-project/ray/pull/15756 added.
